### PR TITLE
Adding minor changes

### PR DESCRIPTION
**Pull request by KilgoreTrout9**
Making current

### CODE REVIEW

File: client/components/login/Login.jsx [hunk 1]
```diff
@@ -78,23 +45,27 @@ const Login = () => {
     setEmail(email);
     setImage(image);
 
+    // get the user document from the database
     axios.get(`/auth/${email}`)
       .then(({ data }) => {
         console.log('user obj from database: ', data);
```
Comment (greta-schock) [inline client/components/login/Login.jsx:51]: Rogue console.log

File: client/components/login/Profile.jsx [hunk 2]
```diff
@@ -84,45 +16,48 @@ const Profile = () => {
...
+    .then(() => {
         userData.smoves.forEach((smove) => {
           if (smove.isCurrentSmove) {
             smove.moveTeam.push(teamEmail)
           };
         })
         axios.patch(`/user/${userData.email}`, {data: userData.smoves})
-          .then(({ data }) => console.log('new team member added: ', data))
-          .catch((err) => console.log('error in patch request to add user: ', err))
+        .then(({ data }) => console.log('new team member added: ', data))
```
Comment (greta-schock) [inline client/components/login/Profile.jsx:52]: Do you want to keep this console.log?

Comment (KilgoreTrout9) [reply client/components/login/Profile.jsx:52]: willl change

**Review by mingwencode — APPROVED** [review]

**Review by greta-schock — APPROVED** [review]

**Review by jessesmith-13 — APPROVED** [review]
lgtm

**Pull request merged**